phpstan-baseline.neon

```
@@ -14,8 +14,3 @@ parameters:
             message: "#^Access to an undefined property PhpParser\\\\Node\\:\\:\\$args\\.$#"
             count: 1
             path: src/Visitor/Php/Symfony/AbstractFormType.php
-
-        -
-            message: "#^Access to an undefined property PhpParser\\\\Node\\\\Expr\\:\\:\\$value\\.$#"
```

1 ignored error less! :tada: